server/config/options.go

```diff
@@ -157,17 +162,42 @@ var DefaultConfig = Config{
 		EnableHeap: true,
 	},
 	Quota: QuotaConfig{
-		Enabled:                   false,
-		RateLimit:                 1000,        // requests per second both reads and writes
-		WriteThroughputLimit:      10000000,    // bytes per second
-		ReadThroughputLimit:       10000000,    // bytes per second
-		DataSizeLimit:             10000000000, // bytes
-		LimitUpdateInterval:       5,           // seconds
-		TenantSizeRefreshInterval: 60,          // seconds
-		AllTenantsRefreshInternal: 300,         // seconds
+		Node: LimitsConfig{
+			Enabled: false,
+
+			ReadRateLimit:        1000,     // read requests per second per namespace
+			ReadThroughputLimit:  10000000, // read bytes per second
+			WriteRateLimit:       1000,     // write requests per second
+			WriteThroughputLimit: 10000000, // write bytes per second
+		},
+		Namespace: NamespaceLimitsConfig{
+			Default: LimitsConfig{
+				Enabled: false,
+
+				ReadRateLimit:        1000,     // read requests per second per namespace
+				ReadThroughputLimit:  10000000, // read bytes per second
+				WriteRateLimit:       1000,     // write requests per second
+				WriteThroughputLimit: 10000000, // write bytes per second
+			},
+			Node: LimitsConfig{
+				ReadRateLimit:        1000,     // read requests per second per namespace
+				ReadThroughputLimit:  10000000, // read bytes per second
+				WriteRateLimit:       1000,     // write requests per second
+				WriteThroughputLimit: 10000000, // write bytes per second
+			},
+			Backend:         "datadog",
+			RefreshInterval: 60 * time.Second,
+		},
+		Storage: StorageLimitsConfig{
+			Enabled:         false,
+			DataSizeLimit:   100 * 1024 * 1024,
+			RefreshInternal: 60 * time.Second,
+		},
 	},
 	Observability: ObservabilityConfig{
-		Enabled: false,
+		Provider:    "datadog",
+		ProviderUrl: "https://us3.datadoghq.com",
```

Comment:
Should this be default configuration?

Reply:
> Should this be default configuration?

Do you mean have empty default instead?

> Maybe if there is no observability configured, only do local limits.

Good point. Changed it that way.